### DISCUSSION TITLE
Have beatmap select displayed rank respect selected mods option

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#22444] [Ideas]
Have the little rank icon displayed on beatmaps in the beatmap select menu show the best rank achieved with the currently selected mods if the selected mods option is enabled on the leaderboard. So for example, let's say someone has a D rank nomod play on a map with 350,000 points and an S rank relax play on the same map with 98,000 points. In the current implementation, the D rank score is higher, so it will always display a D rank on the beatmap in the menu. However, my proposal is that it would instead display an S rank on the beatmap if the selected mods option is enabled and the relax mod is selected. I'm neutral about whether it should display no rank or follow the current behavior if there are no plays set on the beatmap with the currently selected mods.

**Comment by Rekunan** [comment]
If I'm understanding and remembering correctly, you'll be able to achieve this in the future when they add the ability to sort the leaderboard by achieved acc.
